File-Storage-Security/Post-Scan-Action-Automations/aws-slack-notification-automation/ReadMe.md

```diff
@@ -0,0 +1,32 @@
+# Subscribe deployed FSS Storage Stack SNS topics to Slack Notification Plugin.
+This script will subscribe a deployed Slack plugin to all FSS storage stacks. After deployment any malicious event will be sent to the defined slack channel. 
+
+**Before you deploy**
+
+   * Install supporting tools
+        - Install the AWS command line interface (CLI). 
+        See [Installing the AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) for details.
+
+   * If not already present, [deploy the Slack Plugin for File Storage Security](https://github.com/trendmicro/cloudone-filestorage-plugins/tree/master/post-scan-actions/aws-python-slack-notification).
+  
+  * Obtain the following parameters.
+      - API Key - Generate a [Cloud One API Key](https://cloudone.trendmicro.com/docs/account-and-user-management/c1-api-key/)
+      - Cloud One Account Region - example: ```us-1```.
```

Comment:
Will be helpful to our people overseas to reference all the regions with this link -> https://cloudone.trendmicro.com/docs/identity-and-account-management/c1-regions/

So they don't need to find the other regions themselves.

Reply:
Added that in README